packages/avatars/src/elements/Avatar.tsx

```diff
@@ -79,7 +79,7 @@ const AvatarComponent = forwardRef<HTMLElement, IAvatarProps>(
             type={computedStatus}
             surfaceColor={surfaceColor}
             aria-label={statusLabel}
-            role="img"
+            as="figcaption"
```

Comment:
@geotrev without the deployment to play with, I'm not clear on the potential impact (although I like the direction). Can you comment on the change? Also, please reference https://github.com/zendeskgarden/react-components/pull/1403#discussion_r958903497 (tbh, the entire discussion thread might be helpful). This one got kicked around a lot with @Francois-Esquire, so I want to be sure we have the a11y nailed down. :pray:

Reply:
Ah, yes, so sorry. I'll update the description so this change is clearer.

Reply:
I updated the description. I meant to actually ask for input form y'all because it is a slight change in SR content. Basically reads the status before focusing the image of the `figure`, which if I recall is how `figure` content is normally read (compared against [MDNs example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)).

Thanks for catching this!